.eslintignore

```diff
@@ -0,0 +1,3 @@
+build
+dist
+node_modules
```

Comment:
Does this file also support the `/` prefix to indicate the names are directories?

`docs/` should also be excluded.

Reply:
I believe it does.